Cargo.toml

```diff
@@ -70,3 +70,4 @@ pkcs11-provider = ["pkcs11", "picky-asn1-der", "picky-asn1", "picky-asn1-x509",
 tpm-provider = ["tss-esapi", "picky-asn1-der", "picky-asn1", "picky-asn1-x509"]
 all-providers = ["tpm-provider", "pkcs11-provider", "mbed-crypto-provider"]
 docs = ["pkcs11-provider", "tpm-provider", "tss-esapi/docs", "mbed-crypto-provider"]
+peer-credentials-authenticator = []
```

Comment:
See [my related comment](https://github.com/parallaxsecond/parsec-book/pull/57#discussion_r485089884), if it is agreed, I would personally prefer all occurences of "peer credentials" to be renamed "Unix peer credentials".